storage/Cargo.toml

```diff
@@ -11,3 +11,6 @@ move-core-types.workspace = true
 move-table-extension.workspace = true
 moved.workspace = true
 rocksdb.workspace = true
+
+[dev-dependencies]
+hex-literal = "0.4"
```

Comment:
nit: probably should move to workspace

Reply:
As far as I can tell, we were already using `alloy::hex`, at least in tests. Seems to be the same functionality

Reply:
It's the same library that `alloy` uses (`alloy::hex` is a re-export from `hex-literal`), so this will not add any additional work to the compiler, and we don't need to bring in entire `alloy`.